.github/workflows/ci.yml

```diff
@@ -36,9 +36,22 @@ jobs:
       - name: Run benchmarks
         run: npm run bench
 
+  attw:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - run: npm ci
+      - run: npm run attw
+
   lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
```

Comment:
very surprised that linting and npm was working without this. i guess all boxes have node by default?

Reply:
yeah i don't ever do this in linting/etc steps b/c the node.js version isn't really a factor here, but up to you on if we should leave this in. the actions runner ships with an LTS version by default: https://github.com/actions/runner-images/blob/ubuntu24/20250223.1/images/ubuntu/Ubuntu2404-Readme.md